components/neutron/aio-values.yaml

```diff
@@ -12,6 +12,39 @@ endpoints:
     hosts:
       default: rabbitmq-nodes
 
+network:
+  # we're using ironic and actual switches
+  backend:
+    - baremetal
+
+  # configure OpenStack Helm to use Undercloud's ingress
+  # instead of expecting the ingress controller provided
+  # by OpenStack Helm
+  use_external_ingress_controller: true
+
+conf:
+  plugins:
+    ml2_conf:
+      ml2:
+        # set the default ml2 backend as the logger for testing, actual deployment should override
+        mechanism_drivers: logger
+  neutron:
+    DEFAULT:
+      # we aren't using neutron routers
+      service_plugins: ""
+      # we don't want HA L3 routers
+      l3_ha: False
```

Comment:
yaml lint doesn't like this... should this just be `false`? Or maybe `"False"`?

Reply:
Yeah you're right. It's parsed in Python it turns out so it needed to be `"False"` like you suggested.